reporthandling/results/v1/resourcesresults/datastructures.go

```diff
@@ -19,6 +19,8 @@ type Result struct {
 type ResourceAssociatedControl struct {
 	ControlID               string                   `json:"controlID"`
 	Name                    string                   `json:"name"`
+	Status                  apis.StatusInfo          `json:"status"`
+	SubStatus               apis.ScanningSubStatus   `json:"subStatus"`
```

Comment:
We should consider using the object StatusInfo and have all status-related info handled there